services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -51,14 +53,28 @@ const Asset = ({ libraryData }) => {
   const [assetData] = libraryData.assets
   const { name, description, inherits: inheritsData, thumbnailData: imageData } = assetData.content
 
+  const id = libraryData.content.id
+
+  const libraryId = id
+    .replace(/\b(([a-zÁ-ú]){3,})/g, (w) => w.charAt(0).toUpperCase() + w.slice(1))
+    .replace(/-/g, ' ')
+
   const seo = {
     title: name,
-    description
+    description,
+    id: id,
```

Comment:
The `seo` object is used here to pass props into the `NextSeo` component. I don't think we want to add additional props in here like `id` and `libraryId` because that could be used in unexpected ways in the `NextSeo` component?

https://github.com/garmeeh/next-seo